src/setNextRuntimeConfig.ts

```diff
@@ -0,0 +1,16 @@
+import * as envConfig from 'next/dist/next-server/lib/runtime-config';
```

Comment:
Would it make more sense to keep this function along with the others in one place? (`src/nextConfig`).

Reply:
My take here (I could have misunderstood Next.js), is that `next.config.js` and Next.js runtime configuration (`next/config`) are 2 unrelated moving parts of Next.js.

`next/config` depends on `next.config.js` as much as many other Next.js parts do. On our case we also use `next.config.js` to get the allowed file extensions.

But maybe they are closer than I though and keeping them together might make sense. How do you see it?

Reply:
Agree those are 2 different parts. LGTM